css/interface.css

```diff
@@ -138,28 +134,37 @@ header .title-holder p {
 
 .panel.focus .glyphicon-remove {
   opacity: 0 !important;
-  -webkit-transition: opacity 50ms linear;
+  transition: opacity 50ms linear;
 }
 
 .panel-heading {
   position: relative;
   padding: 0;
 }
 
-.panel-heading:hover {
-  cursor: -webkit-grab;
-  cursor: -moz-grab;
+.grabbable {
```

Comment:
Well, you could've stuck with `.panel-heading`, but I appreciate the reusable `.grabbale` class ☺️